internal/security/secret.go

```diff
@@ -33,7 +33,7 @@ import (
 
 // SecretProvider cache storage for the secrets
 type SecretProvider struct {
-	secretClient  pkg.SecretClient
+	SecretClient  pkg.SecretClient
```

Comment:
Why did this need to change? Function doesn't need it as it has GetSecrets() in AppContext.

Reply:
its a shortcut in order to avoid having to refactor a bunch of stuff that should be done in a separate PR. it needs to be mocked, but its in a different package from where i need it, so i need to export it